crates/oxc_linter/src/rules/typescript/prefer_enum_initializers.rs

```diff
@@ -0,0 +1,104 @@
+use oxc_ast::AstKind;
+use oxc_diagnostics::{
+    miette::{self, Diagnostic},
+    thiserror::Error,
+};
+use oxc_macros::declare_oxc_lint;
+use oxc_span::Span;
+
+use crate::{context::LintContext, rule::Rule, AstNode};
+
+#[derive(Debug, Error, Diagnostic)]
+#[error("typescript-eslint(prefer-enum-initializers):")]
+#[diagnostic(severity(warning), help(""))]
```

Comment:
Aha, we need to fill error message (line 12) and help message (line 13).

Reply:
ugh, I keep forgetting those. thanks, fixed!